ibc/store/bulk_store_cache.go

```diff
@@ -0,0 +1,187 @@
+package store
+
+import (
+	"fmt"
+	"sync"
+
+	"github.com/pokt-network/pocket/persistence/kvstore"
+	"github.com/pokt-network/pocket/runtime/configs"
+	coreTypes "github.com/pokt-network/pocket/shared/core/types"
+	"github.com/pokt-network/pocket/shared/modules"
+	"github.com/pokt-network/pocket/shared/modules/base_modules"
+)
+
+var (
+	_         modules.BulkStoreCacher = &bulkStoreCache{}
+	cacheDirs                         = func(storesDir string) string { return fmt.Sprintf("%s/caches", storesDir) }
```

Comment:
Nit: this space is super weird to read, consider spacing them out to avoid the formatting change or making them individual var declarations instead of grouping them together.